test/Project.toml

```diff
@@ -0,0 +1,10 @@
+[deps]
+LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
+PGFPlotsX = "8314cec4-20b6-5062-9cdb-752b83310925"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+TikzPictures = "37f6aa50-8035-52d0-81c2-5a1d08754b2d"
+UnicodePlots = "b8865327-cd53-5732-bb35-84acbb429228"
+
+[compat]
+LaTeXStrings = "^1.3"
```

Comment:
Could you add the missing entries here?

Reply:
Sure. I was wondering what default versions are used for the [extra] entry of the main Project.toml, since the tests were working with whatever these versions were before I added the requirement feature ?

Reply:
https://github.com/JuliaSmoothOptimizers/BenchmarkProfiles.jl/actions/runs/6907307413/job/18794184718?pr=110#step:6:46

It will take the most up-to-date version

Reply:
Should compat for test.jl be added there too ?